lib/gui/os/window-progress.js

```diff
@@ -59,4 +97,5 @@ exports.clear = () => {
   const ELECTRON_PROGRESS_BAR_RESET_VALUE = -1
 
   exports.currentWindow.setProgressBar(ELECTRON_PROGRESS_BAR_RESET_VALUE)
+  exports.currentWindow.setTitle(initialTitle)
```

Comment:
I feel like the cons outweigh the pros here unless I'm missing something, we're basically just moving the logic that's already present in the separation of `set` and `clear` into `makeTitle` by adding an if-statement.